install-guide.md

```diff
@@ -88,83 +88,33 @@ wsl --install -d Ubuntu
 
 Setup user credentials
 
-**Configure Ubuntu and build RPCSX:**
-  Add the PPA repository that has the packages we need
+**Download and install RPCSX:**
+  
+  ```sh
+  wget -O - https://raw.githubusercontent.com/DHrpcs3/rpcsx-setup-wsl/refs/heads/main/rpcsx-setup-wsl.sh | bash
```

Comment:
Do we really want to recommend a curl-bash pattern? Why not clone the repo and run it?

Reply:
Yes. Git need to be installed, need to teach users how to use git to update this script. This instruction is not for developers. Lets keep things simple for windows users that just want to test emulator

Reply:
Changing this would change 1 command to 5 or 6.
Depending on apt install git or add upstream git PPA then apt install git,
Clone the repo, change the directory, Make the script executable, and execute the script.